types/webpack/index.d.ts

```diff
@@ -580,7 +580,7 @@ declare namespace webpack {
         }
         interface SplitChunksOptions {
             /** Select chunks for determining shared modules (defaults to \"async\", \"initial\" and \"all\" requires adding these chunks to the HTML) */
```

Comment:
I wasn't sure whether or not to add extra documentation here, since this looks to be straight from the webpack config schema